keras/legacy/interfaces.py

```diff
@@ -80,3 +87,11 @@ def raise_duplicate_arg_error(old_arg, new_arg):
     conversions=[('pool_length', 'pool_size'),
                  ('stride', 'strides'),
                  ('border_mode', 'padding')])
+
+legacy_pooling2d_support = generate_legacy_interface(
+    allowed_positional_args=['pool_size', 'strides', 'padding'],
+    conversions=[('pool_length', 'pool_size'),
+                 ('stride', 'strides'),
+                 ('border_mode', 'padding'),
+                 ('dim_ordering', 'data_format')],
+    value_conversions={'dim_ordering': [('tf', 'channels_last'), ('th', 'channels_first'), ('default', None)]})
```

Comment:
You can make this a dict instead of a list of tuples, that would be more natural. In this case, the order of the values does no matter.